webapp/platform/components/src/generic_modal/generic_modal.tsx

```diff
@@ -199,7 +201,7 @@ export class GenericModal extends React.PureComponent<Props, State> {
                             </>
                         )}
                     </Modal.Header>
-                    <Modal.Body>
+                    <Modal.Body className={classNames({divider: this.props.bodyDivider})}>
```

Comment:
I agree with the thinking, and it's quite easy to negate (`... || undefined`), but this is the standard pattern for simple inline `classNames` usage like this—there are hundreds of `className=""` instances in snapshots, so I'm a bit hesitant to make this one different to avoid touching what is only a couple snapshots. 2/5; happy to make the change if you think it's necessary.

Reply:
Nope, I just would have expected it have caused many more issues with snapshots. I guess `generic_modal` isn't used very ofter.